test/test_rubygems_update.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "rubygems/test_case"
+require "open3"
+
+class TestRubygemsUpdate < Minitest::Test
+
+  def test_builds_ok
+    output, status = Open3.capture2e("gem build rubygems-update.gemspec")
```

Comment:
This will be testing the gem cli installed in the system and not the one in `bin/gem` maybe we want to test this using `ruby -e ...` ?

Reply:
Thanks, fixed!